src/Microsoft.AspNet.Mvc.Core/Resources.resx

```diff
@@ -409,4 +409,7 @@
   <data name="ViewComponent_AmbiguousTypeMatch_Item" xml:space="preserve">
     <value>Type: '{0}' - Name: '{1}'</value>
   </data>
+  <data name="SerializableError_DefaultError" xml:space="preserve">
+    <value>An error has occured.</value>
```

Comment:
Is this the string from the original code? We should probably say something like 'The input was not valid' since this is specifically for validation errors.